### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by martint**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.101.rst [hunk 1]
```diff
@@ -58,3 +61,19 @@ Hive Changes
 * Add JMX stats to PrestoS3FileSystem.
 * Add ``hive.recursive-directories`` config option to recursively scan
   partition directories for data.
+
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.101.rst]: Only one blank line

File: presto-docs/src/main/sphinx/release/release-0.101.rst [hunk 2]
```diff
@@ -58,3 +61,19 @@ Hive Changes
...
+
+SPI Changes
+-----------
+
+* Add connector callback for rollback of ``INSERT`` and ``CREATE TABLE AS``.
+* Introduce an abstraction for representing physical organizations of a table
+  and describe properties such as partitioning, grouping, predicate and columns.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.101.rst]: This reads weird. I think it should be "describing": "an abstraction for representing ... and describing ..."

File: presto-docs/src/main/sphinx/release/release-0.101.rst [hunk 3]
```diff
@@ -58,3 +61,19 @@ Hive Changes
...
+  The ConnectorPartition and related interfaces are deprecated and will be removed
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.101.rst]: ``ConnectorPartition``

File: presto-docs/src/main/sphinx/release/release-0.101.rst [hunk 4]
```diff
@@ -58,3 +61,19 @@ Hive Changes
...
+  in a future version.
+* Rename ConnectorColumnHandle to ColumnHandle.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.101.rst]: * Rename ``ConnectorColumnHandle`` to ``ColumnHandle``.

**Comment by electrum** [comment]
Looks good

File: presto-docs/src/main/sphinx/release/release-0.101.rst (hunk 3 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.101.rst]: Remove leading "The"

File: presto-docs/src/main/sphinx/release/release-0.101.rst (hunk 2 above)
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.101.rst]: Ah, yes, I refactored the sentence a bit after I wrote the initial version and missed that

**Pull request merged**